### DISCUSSION TITLE
Downgrade von K3663_1/K3663 to K2589

### DISCUSSION THREAD

**Discussion by johunb** [Mr-MIBonk/M.I.B._More-Incredible-Bash#79] [Q&A]
In den AIO-File MHI2_ER_AU57x_K3663_1_MU1425 is a file called K3663-to-K2589-Downgrade-SVM(MHI2ER2589).7z

ReadMe Custom FW Update.txt says:
SVM:
If you have access to official SVM you can use that with SVM code MHI2ER3663
On older models with 00xx and 2xxx versions you can use the "Downgrade" update in the K3663-to-K2589 folder. This will downgrade all needed modules to be able to pass SVM with code MHI2ER2589, all activations and features remain.
After SVM you will have to enable Carplay and AA again in adaptions!! Unit will get new Calibration from SVM, FECs should remain.

What do I have to do, if  want to downgrade from K3663_1/K3663 to K2589? I come from MHI2_ER_AUG11_S0034_MU0073.

Thank you!

**Comment by harman-f** [comment]
Not 100% the topic for this repo ;)

Still, get these files and put them on root of a freshly FAT32 formatted SD card:
![image](https://user-images.githubusercontent.com/98130152/209378182-5f27e33e-a19b-4345-87c6-93979a5658d5.png)

Install like a FW update.

**Comment by harman-f** [comment]
For more supprt go here: https://t.me/+T_pADdwJyUXq0VoH
And look for https://t.me/DukeFawks
